test/ReadsUtils_server_test.py

```diff
@@ -1050,25 +1103,11 @@ def test_upload_fail_interleaved_for_single(self):
              'fwd_id': ret['id'],
              'name': 'bar'
              },
-            'Invalid fasta file /kb/module/work/tmp/fwd/Sample5_interleaved' +
-            '.fastq from Shock node ' + ret['id'])
+            'Invalid fasta file /kb/module/work/tmp/fwd/Sample5_interleaved.fastq')
+#            'Invalid fasta file /kb/module/work/tmp/fwd/Sample5_interleaved' +
+#            '.fastq from Shock node ' + ret['id'])
         self.delete_shock_node(ret['id'])
 
-    def test_upload_fail_interleaved_for_paired(self):
-        ret1 = self.upload_file_to_shock('data/Sample1.fastq')
-        ret2 = self.upload_file_to_shock('data/Sample5_interleaved.fastq')
-        self.fail_upload_reads(
-            {'sequencing_tech': 'tech',
-             'wsname': self.ws_info[1],
-             'fwd_id': ret1['id'],
-             'rev_id': ret2['id'],
-             'name': 'bar'
-             },
-            'Invalid fasta file /kb/module/work/tmp/rev/Sample5_interleaved' +
-            '.fastq from Shock node ' + ret2['id'])
-        self.delete_shock_node(ret1['id'])
-        self.delete_shock_node(ret2['id'])
-
     # Download tests ########################################################
 
```

Comment:
There's still no tests covering the case of providing f and r reads in shock nodes that can't be interleaved; all the possible failure modes for interleaving need to be covered